recipes/xbitinfo/install_julia.patch

```diff
@@ -0,0 +1,204 @@
+diff --color -Naur xbitinfo-0.0.2.orig/setup.py xbitinfo-0.0.2/setup.py
+--- xbitinfo-0.0.2.orig/setup.py	2022-07-11 17:37:19.000000000 -0300
++++ xbitinfo-0.0.2/setup.py	2022-08-29 10:08:18.745686078 -0300
+@@ -5,26 +5,6 @@
+ import os
+ 
+ from setuptools import find_packages, setup
+-from setuptools.command.develop import develop
+-from setuptools.command.install import install
+-
+-julia_install_command = "julia install_julia_packages.jl"
+-
+-
+-class PostDevelopCommand(develop):
+-    """Post-installation for development mode."""
+-
+-    def run(self):
+-        develop.run(self)
+-        os.system(julia_install_command)
+-
+-
+-class PostInstallCommand(install):
+-    """Post-installation for installation mode."""
+-
+-    def run(self):
+-        install.run(self)
+-        os.system(julia_install_command)
```

Comment:
@observingClouds this is not ready yet but there are a few strategies I'd like to discuss with you and, if you agree, upstream them. First one is that I removed the installation at setup time.